anthos-bm-gcp-terraform/terraform.tfvars.sample

```diff
@@ -6,6 +6,7 @@ abm_cluster_id                = "cluster1"
 anthos_service_account_name   = "baremetal-gcr"
 resources_path                = "./resources"
 machine_type                  = "n1-standard-8"
+# service_account               = "<GCE service account with iam.serviceAccountKeyAdmin permission>"
```

Comment:
Can we move this below/after `instance_count` - so that all the commented variables are by default stacked close to each other. Also from [this comment](https://github.com/GoogleCloudPlatform/anthos-samples/pull/120/files#r787953957) we might want to update the variable name here to `gce_vm_service_account`